docker/Dockerfile.gpu

```diff
@@ -0,0 +1,149 @@
+    export PATH=/usr/local/cuda/bin:$PATH && \
+    export PKG_CONFIG_PATH=$HOME/compiled/lib/pkgconfig && \
+    cd "$HOME/ffmpeg" && \
+    make && make install
+
+RUN export PATH=$HOME/compiled/bin:$PATH && \
+    export PATH=/usr/local/cuda/bin:$PATH && \
+    export PKG_CONFIG_PATH=$HOME/compiled/lib/pkgconfig && \
+    go get github.com/livepeer/go-livepeer/cmd/livepeer && \
+    cd "$HOME/go/src/github.com/livepeer/go-livepeer" && \
+    go build ./cmd/livepeer/livepeer.go && \
+    go build ./cmd/devtool/devtool.go
+
+ENV LD_LIBRARY_PATH /usr/local/lib/
+
+# final stage
+FROM nvidia/cuda:10.1-devel-ubuntu18.04
+ENV DEBIAN_FRONTEND noninteractive
+RUN export DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get dist-upgrade -y && \
+    apt-get install software-properties-common -y && \
+    add-apt-repository ppa:longsleep/golang-backports && \
+    apt-get update && apt-get dist-upgrade -y && \
+    apt-get install -y build-essential pkg-config autoconf gnutls-dev \
+            git curl golang-go && \
+    ##apt-get install -y libnvidia-encode-430 && \
+    ##curl -O http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/cuda-repo-ubuntu1804_10.0.130-1_amd64.deb && \
+    ##dpkg --force-confdef --force-confnew -i ./cuda-repo-ubuntu1804_10.0.130-1_amd64.deb && \
+    ##apt-key adv --fetch-keys http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/7fa2af80.pub && \
+    
+    apt-get update && \
+    #apt-get install cuda-10-1 -y && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+RUN add-apt-repository universe && apt-get update &&  apt-get install -y jq
+
+RUN curl -O http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/cuda-repo-ubuntu1604_10.1.168-1_amd64.deb && \
+    dpkg --force-confdef --force-confnew -i ./cuda-repo-ubuntu1604_10.1.168-1_amd64.deb && \
+    apt-key adv --fetch-keys http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/7fa2af80.pub && \
+   
+    curl -O http://ftp.ubuntu.com/ubuntu/ubuntu/pool/restricted/n/nvidia-graphics-drivers-418/libnvidia-compute-418_418.56-0ubuntu1_amd64.deb && \
+    dpkg --force-confdef --force-confnew -i ./libnvidia-compute-418_418.56-0ubuntu1_amd64.deb && \
+
+    curl -O http://ftp.ubuntu.com/ubuntu/ubuntu/pool/restricted/n/nvidia-graphics-drivers-418/libnvidia-decode-418_418.56-0ubuntu1_amd64.deb && \
+    dpkg --force-confdef --force-confnew -i ./libnvidia-decode-418_418.56-0ubuntu1_amd64.deb && \
+
+    curl -O http://ftp.ubuntu.com/ubuntu/ubuntu/pool/restricted/n/nvidia-graphics-drivers-418/libnvidia-encode-418_418.56-0ubuntu1_amd64.deb && \
+    dpkg --force-confdef --force-confnew -i ./libnvidia-encode-418_418.56-0ubuntu1_amd64.deb
+
+# everything above this line has already been cached in the build stage -
+# we save time and disk space by keeping the common parts of each stage
+# identical at the start of each image build
```

Comment:
This is what I was talking about in my previous comment.    I'm not sure this comment is in the correct location -- it should be moved up, but only if the first few commands are identical to those in the build stage, otherwise remove the comment.